app/workers/import_completion_denylist_worker.rb

```diff
@@ -0,0 +1,17 @@
+class ImportCompletionDenylistWorker
+  LOCK_NAME = "import_completion_denylist_worker".freeze
+
+  include Sidekiq::Worker
+
+  def perform
+    CompletionDenylistEntry.with_advisory_lock(LOCK_NAME) do
+      client.import(CompletionDenylistEntry.all)
```

Comment:
Ok, so here, `all` is being passed in! So why is `all` being called from within the `import` method as well?